business-central-parent/business-central-webapp/pom.xml

```diff
@@ -2708,6 +2708,9 @@
             <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-forms-client</compileSourcesArtifact>
             <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-project-api</compileSourcesArtifact>
             <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-project-client</compileSourcesArtifact>
+	    <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-kogito-api</compileSourcesArtifact>
+            <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-kogito-client</compileSourcesArtifact>
+            <compileSourcesArtifact>org.kie.workbench:kie-wb-common-kogito-client</compileSourcesArtifact>
```

Comment:
why this needs to be added in BC? Is that API used in 7.x applications? I would expect it is in kogito editors not in our 7.x stream as the name says.

Reply:
The refactoring of Stunner to support re-use of the _core_ editors in both Business Central and kogito (VSCode etc plugins) necessitated the introduction of new modules. The source code for the new modules needs to be declared to the GWT compiler in order for GWT compilation to succeed.

Reply:
maybe you should used different naming then if that is general API for using by both code bases

Reply:
I'm agree with @mareknovotny guys - I think kogito should be used for the impls rather than for APIs... 

At least in Stunner we'd been always making the difference between: "standalone" and "project" stuff. For the implementation, most of the "project" related class names start  with or contains some "Project" term, and the "standalone" ones are usually named using the term "Default" or "Standalone"... 

So this way, in Stunner terms, the term "kogito" can be directly mapped to the "standalone" stuff, but not at API level, IMO at the implementation. 

This is why I was proposing also to remove the "stunner-kogito-api" module and refactor that code into the core, it's fine then having the kogito, standalone, project or whatever impl on top, but IMO that API should be integrated into the core stuff

Just leaving my 2 cents here too guys, I don't way it's a blocker, but as far as we introduce something into `master` it gets visibility, so I would prefer to have this discussions now rather than later...

Thx!

Reply:
@manstis I read @romartin 's [comment](https://github.com/kiegroup/kie-wb-distributions/pull/965#discussion_r330814343) and have feeling that in Stunner they use term `standalone` for client side marshalling version, while in Dmn we use `standalone` for server side marshalling version? Do I miss something?

Reply:
@romartin Can you please clarify your terms?

DMN and scesim use `-runtime` for kogito standalone and `-testing` for kogito with wrapper (for development). We use the term `-standalone` for what was the _project_ showcase.

Reply:
hey @manstis @jomarko 
In stunner there are two environments on top the core/commons libraries, so two showcaes too - the `standalone` and the `project` ones.

The term `standalone` or `project` has nothing to do with BPMN marshallers - those marshallers are either on bpm-backend (server) or bpmn-marshalling modules (client), and then, on of top those modules, we'll have in kogito 3 environments now:
* `standalone` -> a webapp, with as less kie deps as possible, but not as kogito. It still uses the BPMN server sdie marshallers
* `project` -> the webapp for the whole kie environment. It still uses the BPMN server sdie marshallers
* `bpmn-runtime-webapp` -> the webapp for kogito's BPMN editor. It uses the client side marshallers

So as you can see, the terms `standalone`, `project` or `kogito` are (or should be) top level modules, on top of the core/commons ones for each domain (BPMN,DMN), and in case of BPMN, marshallers are not specifcly tight to any concrete environment, they're just maven deps thta can be added to achieve whatevet kind of marshallers you want.

Hope this clarifies a bit...